sdk/core/azure-core/test/ut/datetime.cpp

```diff
@@ -501,12 +504,12 @@ TEST(DateTime, ParseTimeInvalid2)
       "1971-01-01T00:60:00Z", // minute too big
       "1971-01-01T00:00:70Z", // second too big
       "1971-01-01T00:00:61Z",
-      "1601-01-01T00:00:00+00:01", // time zone underflow
+      "0001-01-01T00:00:00+00:01", // time zone underflow
       // "1970-01-01T00:00:00.Z", // accepted as invalid timezone above
-      "1970-01-01T00:00:00+24:00", // bad tzoffsets
-      "1970-01-01T00:00:00-30:00",
+      //"1970-01-01T00:00:00+24:00", // bad tzoffsets
```

Comment:
Why did we comment out these test cases?

Reply:
Because it is valid offset, per spec, and on .NET